index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>Intention Timer</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.google.com/specimen/Montserrat?query=mont&selection.family=Montserrat:300,400" rel="stylesheet">
+  </head>
+
+  <body>
+    <header>
+      <p class="header-title">Intention Timer</p>
+    </header>
+    <main>
+      <section class="left-section">
+        <p class="left-title">New Activity</p>
+        <section class="content-section">
+          <p class="select-category">Select a category:</p>
+          <div class="category-button-section">
+            <button class="category-button">
+              <!-- add in image sources  -->
+              <img src="..." alt="lightbulb over book">
```

Comment:
We'll need to upload all our images into our assets dir and then remember to update these!